lib/buildConfig.js

```diff
@@ -4,6 +4,120 @@ var bowerPath = require('../../bower-discover');
 
 var collectionDir;
 
+var projectObj = {collections: [], menuItems: [], menu: {}, bowerItems: [], bower: {}, cdns: {}};
+
+var parseConfig = function (root, collection, collectionConfig) {
+
```

Comment:
'collection' might be somewhat unclear here - possibly 'collectionName' would be better?